### ISSUE TITLE
Support Markdown image insertion syntax.

### ISSUE THREAD

**Issue by chen-qingyu** [hexojs/hexo#5505]
### Check List

- [X] I have already read [Docs page](https://hexo.io/docs/).
- [X] I have already searched existing issues.

### Feature Request

> Once enabled, an asset image will be automatically resolved to its corresponding post’s path. For example, “image.jpg” is located at “/2020/01/02/foo/image.jpg”, meaning it is an asset image of “/2020/01/02/foo/“ post, `![](image.jpg)` will be rendered as `<img src="/2020/01/02/foo/image.jpg">`.

This makes it impossible to preview images in the local markdown editor like vscode.

Many plugins have been created to solve this problem, such as one I wrote: [hexo-markdown-image](https://github.com/chen-qingyu/hexo-markdown-image).

Hope to native support relative path referencing of images: `![](foo/image.jpg)`.

**Comment by lmliheng** [comment]
Yes, I have also encountered this problem. Why do some themes write the path as dead😥😥

**Comment by jzj1993** [comment]
I have the same needs. You can try this plugin, it should do the image copy work and solve the issue: https://github.com/jzj1993/hexo-relative-post-images

**Comment by stevenjoezhang** [comment]
Please join the discussion here https://github.com/hexojs/hexo-renderer-marked/issues/216
